### DISCUSSION TITLE
Feature request: step timestamp for acceptance test

### DISCUSSION THREAD

**Discussion by ksiv** [Codeception/Codeception#6189] [Ideas]
#### What are you trying to achieve?
trying to locate most time consuming step in acceptance test.
#### What do you get instead?
Whole test execution time

p.s. would be nice to have it just in test output to keep the rest intact. A current timestamp or the time from test start - no matter.

**Comment by stefanfisk** [comment]
Did you ever figure out a good strategy for this?

**Comment by ksiv** [comment]
@stefanfisk oh, my friend, this was an eternity back, I don't even remember the case, but I remember we did a lot of customization locally for  Codeception then moved to some other framework on Java due to Company strategy. This request can be closed, I think. Next time I need it I 'll just add it myself.
